Cargo.toml

```diff
@@ -963,6 +966,11 @@ name = "determine_if_only_one_instance_is_running"
 path = "src/determine_if_only_one_instance_is_running.rs"
 
 [[bin]]
+# http://rosettacode.org/wiki/Digital_root
+name = "digital_root"
+path = "src/digital_root.rs"
```

Comment:
This is the problem here. :) I think you can just remove this block since you have it listed in line 227. :)